compiler/next/test/frontend/testParse.cpp

```diff
@@ -17,12 +17,14 @@
  * limitations under the License.
```

Comment:
Do we want to continue to cram parsing tests into `testParse.cpp` or is it okay is we spread things out into other files as I did for `testParseLocal.cpp` and testParseSerial.cpp`? I'm worried that the file might become a bit too crammed.

Reply:
Yeah, I think I'm going to start adding some other files. In seek of balance.